operate/services/manage.py

```diff
@@ -1491,19 +1491,25 @@ def fund_service_single_chain(  # pylint: disable=too-many-arguments,too-many-lo
                     address=key.address,
                 )
                 self.logger.info(
-                    f"Agent {key.address} Asset: {asset_address} balance: {agent_balance}"
+                    f"[FUNDING_JOB] Agent {key.address} Asset: {asset_address} balance: {agent_balance}"
                 )
                 if agent_fund_threshold > 0:
```

Comment:
Why is this check not performed in safe as well?

Reply:
I don't really see why we have this check. agent_fund_threshold will always be passed as positive without any programming error

Reply:
Me neither. Assuming the template has nonnegative values for the fund requirement, if we set, e.g. 0 fund requirements for USDC, then the condition agent_balance < agent_fund_threshold will not be triggered anyways...